core/src/main/java/com/radixdlt/messaging/core/MessageCentralImpl.java

```diff
@@ -144,7 +144,11 @@ public MessageCentralImpl(
     this.outboundThreadPool =
         new SimpleThreadPool<>(
             "Outbound message processing",
-            1, // Ensure messages sent in-order
+            /* This was meant to ensure messages are sent in order,
```

Comment:
:D - how did you discover this? And more importantly - what impact does this have?

Reply:
Also - does this mean we only ever send 1 message at any time?